import_data.py

```diff
@@ -0,0 +1,312 @@
+# Imports a tab delimited data file containing
+# reactor data to output formatted input for
+# Cycamore Reactors.
+from pyne import nucname as nn
+import csv
+import jinja2
+import os
+import sys
+
+
+def import_csv(in_csv, delimit):
```

Comment:
Yes, but this script uses it more heavily than analysis.py file.

I'll import this script in analysis.py and remove the function from analysis.py instead.